pkg/network/ebpf/c/protocol-classification-helpers.h

```diff
@@ -0,0 +1,207 @@
+#ifndef __PROTOCOL_CLASSIFICATION_HELPERS_H
+#define __PROTOCOL_CLASSIFICATION_HELPERS_H
+
+#include <linux/types.h>
+
+#include "protocol-classification-defs.h"
+#include "protocol-classification-maps.h"
+#include "ip.h"
+
+// Patch to support old kernels that does not support bpf_skb_load_bytes, by adding a dummy implementation.
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 5, 0)
+long bpf_skb_load_bytes(const void *skb, u32 offset, void *to, u32 len) {return 0;}
```

Comment:
Is this code going to run on < 4.5?

Reply:
No because im not activating the hook for such kernels
Just needed the declaration for compilation